AEPIdentity/Sources/MobileIdentities.swift

```diff
@@ -49,7 +49,10 @@ struct MobileIdentities: Codable {
     static let NAMESPACE_ANALYTICS_AID = "AVID"
     static let NAMESPACE_USERIDENTIFIER = "vid"
     static let NAMESPACE_AUDIENCE_UUID = "0"
+    static let NAMESPACE_TARGET_TNTID = "tntid"
```

Comment:
nit: we have `TNT_ID` and `THIRD_PARTY_ID` constants in `IdentityConstants` but `...TNTID` and `...THIRDPARTYID` in this file.

let's get consistent about whether these should have underscores or not :) 

(my vote is yes)

Reply:
Yeah, that's how it was in the cpp implementation. I thought I'd keep these constants same to make it easier to refer back to. It's the same with `NAMESPACE_USERIDENTIFIER` here and `VISITOR_IDENTIFIER` in IdentityConstants for Analytics.